app/web/package.json

```diff
@@ -1,155 +1,96 @@
-    "build": "cross-env CI=true react-scripts build",
-    "postbuild": "react-snap",
-    "test": "cross-env TZ=UTC react-scripts test",
-    "eject": "react-scripts eject",
-    "format": "prettier --write src/*",
-    "lint": "eslint src/",
-    "test-ci": "tsc && cross-env CI=true TZ=UTC react-scripts test --coverageReporters=\"text\" --coverageReporters=\"cobertura\" --coverageReporters=\"lcov\" --reporters=\"default\" --reporters=\"jest-junit\" --coverage",
-    "jest": "jest --modulePaths=src",
-    "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "cross-env NODE_PATH=. build-storybook -s public",
-    "upload-sourcemap": "node sentry.js",
-    "typecheck": "tsc"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "prettier"
-    ],
-    "plugins": [
-      "simple-import-sort"
-    ],
-    "rules": {
-      "simple-import-sort/imports": "warn",
-      "simple-import-sort/exports": "warn",
-      "sort-imports": "off",
-      "import/order": "off",
-      "import/first": "error",
-      "import/newline-after-import": "error",
-      "import/no-duplicates": "error",
-      "import/no-relative-parent-imports": "error"
-    },
-    "overrides": [
-      {
-        "files": [
-          "**/*.ts?(x)"
-        ],
-        "rules": {
-          "@typescript-eslint/no-unused-vars": [
-            "warn",
-            {
-              "argsIgnorePattern": "theme",
-              "varsIgnorePattern": "classes|useStyles"
-            }
-          ]
-        }
-      }
-    ]
-  },
-  "eslintIgnore": [
-    "src/proto/**"
-  ],
-  "browserslist": {
-    "production": [
-      ">0.1%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+    "react-query": "^3.31.0",
+    "seamless-scroll-polyfill": "^2.1.5"
   },
   "devDependencies": {
+    "@babel/core": "^7.16.5",
     "@sentry/cli": "^1.71.0",
-    "@storybook/addon-actions": "^6.1.15",
-    "@storybook/addon-essentials": "^6.1.15",
-    "@storybook/addon-links": "^6.1.21",
-    "@storybook/node-logger": "^6.1.21",
-    "@storybook/preset-create-react-app": "^3.2.0",
-    "@storybook/react": "^6.1.15",
+    "@storybook/addon-actions": "^6.4.9",
+    "@storybook/addon-essentials": "^6.4.9",
+    "@storybook/addon-links": "^6.4.9",
+    "@storybook/builder-webpack5": "^6.4.9",
+    "@storybook/cli": "^6.4.9",
+    "@storybook/manager-webpack5": "^6.4.9",
+    "@storybook/react": "^6.4.9",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/react-hooks": "^7.0.2",
-    "@testing-library/user-event": "^13.5.0",
+    "@testing-library/user-event": "^13.4.1",
     "@types/css-mediaquery": "^0.1.1",
     "@types/geojson": "^7946.0.8",
     "@types/google-protobuf": "^3.15.5",
     "@types/jest": "^26.0.24",
-    "@types/node": "^14.18.0",
+    "@types/node": "16.11.6",
```

Comment:
Have we upgraded to Node 16 on Vercel?

Also, loosely related, do we not need a Dockerfile for this anymore?

Reply:
No, not sure why I changed that